File: src/uk/dwelsh/kaffee/models/Session.java
package uk.dwelsh.kaffee.models;

public class Session {
    private String token;
    private String refreshToken;

    public Session(String token, String refreshToken) {
        this.token = token;
        this.refreshToken = refreshToken;
    }

    public String getToken() {
        return token;
    }

    @Override
    public String toString() {
        return "Session{" +
                "token='" + token + '\'' +
                ", refreshToken='" + refreshToken + '\'' +
                '}';
    }
}

File: src/main/kotlin/uk/dwelsh/kaffee/models/Session.kt
package uk.dwelsh.kaffee.models

class Session(val token: String, private val refreshToken: String) {

    override fun toString(): String {
        return "Session{" +
                "token='" + token + '\'' +
                ", refreshToken='" + refreshToken + '\'' +
                '}'
    }

}